topics/digital-humanities/tutorials/open-refine-tutorial/tutorial.md

```diff
@@ -0,0 +1,473 @@
+---
+layout: tutorial_hands_on
+title: OpenRefine Tutorial for researching cultural data
+level: Introductory
+zenodo_link: 'https://doi.org/10.5281/zenodo.17047254'
+questions:
+- How to use OpenRefine in Galaxy to clean your data? 
+- How to use a workflow in Galaxy to extract and visualise information from your data?
+objectives:
+- Start OpenRefine as an Interactive Tool in Galaxy
+- Use OpenRefine to clean your data (remove duplicates, separate multiple values from the same field, etc.)
+- Export your cleaned data from OpenRefine to Galaxy
+- Use a pre-existing workflow in Galaxy to extract specific information and visualise your findings
+time_estimation: 2H
+key_points:
+- You can use OpenRefine online through Galaxy.
+- OpenRefine allows you to work interactively with messy data.
+- Galaxy allows you to run workflows with your data.
+- With Galaxy, you can visualise your data in various ways.
+contributions:
+  authorship:
+    - dianichj
+    - dadrasarmin
+    - Sch-Da
+  funding:
+    - nfdi4culture
+answer_histories:
+  - label: "UseGalaxy.eu"
+    history: https://usegalaxy.eu/u/armin.dadras/h/visualise-amount-of-objects-in-museum-collection
+    date: 2025-09-19
+---
+# Background
```

Comment:
You will get nicer link previews (e.g. when you share a link to the tutorial on chat or social media) if you don't start it with a header

```suggestion
```